tests/test_double_blind_conference.py

```diff
@@ -2037,9 +2037,9 @@ def test_release_accepted_notes_without_revealing_authors(self, client, request_
         assert tabs
         with pytest.raises(NoSuchElementException):
             notes_panel.find_element_by_class_name('spinner-container')
-        assert tabs.find_element_by_id('accepted-poster-papers')
+        # assert tabs.find_element_by_id('accepted-poster-papers')
```

Comment:
why did you comment out this? can you modify the test and assert the new element id?

Reply:
so these two tab names are correct but they don't appear because there are no papers with these decisions. I can remove the lines instead

Reply:
maybe we should update the webfield and always show all the tabs even if there is not data. What do you think?